components/contribution-flow/FeesOnTopInput.js

```diff
@@ -113,7 +113,7 @@ const FeesOnTopInput = ({ currency, amount, quantity, fees, onChange }) => {
       </P>
       <Flex justifyContent="space-between" flexWrap={['wrap', 'nowrap']}>
         <Flex alignItems="center">
-          <Illustration />
+          <Illustration alt="Thank You Illustration" />
```

Comment:
Generally, and in such cases (illustrations), it's better to have an empty `alt` than have a description that doesn't give any value for accessibility.

Reply:
Agreed, thanks for pointing it out. I've removed those that just describe the image. 😄